packages/rule-no-p3p/README.md

```diff
@@ -0,0 +1,25 @@
+# no-p3p (`no-p3p`)
+
+Don't use P3P related headers
+
+## Why is this important?
+
+P3P -- Platform for Privacy Preferences Project -- is an outdated technology
+that is meant to allow browsers to programmatically check privacy policies. In
+case it's not kept in check with the actual privacy policies, it's a legal risk.
+
+## What does the rule check?
+
+It disallows the "P3P" header.
+
+### Examples that **trigger** the rule
+
+Any resource that has a "P3P" header. Contents of the header are not checked.
```

Comment:
Can you add some examples like the ones in https://github.com/sonarwhal/sonarwhal/tree/master/packages/rule-http-cache#examples-that-trigger-the-rule with examples of the headers received?

Because we are not checking for the content of the header it will be great if you could add an example with some "invalid" content.